front/app/containers/Admin/communityMonitor/CommunityMonitorFormBuilder/index.tsx

```diff
@@ -34,6 +35,15 @@ const CommunityMonitorSurveyFormBuilder = () => {
           ...communityMonitorConfig,
           formCustomFields,
           goBackUrl: `/admin/community-monitor/settings`,
+          getUserFieldsNotice: () => {
+            return phase.data.attributes.user_fields_in_form ? (
+              <UserFieldsInFormNotice
+                projectId={projectId}
+                phaseId={phase.data.id}
+                communityMonitor={true}
+              />
+            ) : null;
+          },
```

Comment:
I think it would be cleaner to add this into the `communityMonitorConfig`/`FormBuilderConfig` like `getAccessRightsNotice` or `getWarningNotice` :thinking: Would that be possible?

Reply:
Have changed this as requested - also changed for native survey